jobsdb/jobsdb.go

```diff
@@ -1598,7 +1597,7 @@ func (jd *HandleT) createDSInTx(tx *sql.Tx, newDS dataSetT, l lock.DSListLockTok
 	return nil
 }
 
-func (jd *HandleT) setSequenceNumberInTx(tx *sql.Tx, l lock.DSListLockToken, newDSIdx string) error {
+func (jd *HandleT) setSequenceNumberInTx(tx *sql.Tx, _ lock.DSListLockToken, newDSIdx string) error {
```

Comment:
we are missing an assertion here that `l` is not `nil`

Reply:
Do you mean this method should check that `l` is not `nil` instead of ignoring it?